tests/integration/cattletest/core/test_agent_delegate.py

```diff
@@ -21,38 +21,39 @@ def ihn_network(admin_client):
     return network
 
 
-def test_delegate_agent_create(client, admin_client, sim_context,
+def test_delegate_agent_create(client, internal_test_client, sim_context,
                                system_account):
-    network = create_and_activate(admin_client, 'hostOnlyNetwork',
+    network = create_and_activate(internal_test_client, 'hostOnlyNetwork',
                                   dynamicCreateVnet=True,
                                   isPublic=True)
-    create_and_activate(admin_client, 'subnet',
+    create_and_activate(internal_test_client, 'subnet',
                         networkId=network.id,
                         networkAddress='192.168.1.0')
-    ni = create_and_activate(admin_client, 'agentInstanceProvider',
+    ni = create_and_activate(internal_test_client, 'agentInstanceProvider',
                              networkId=network.id,
                              agentInstanceImageUuid=_IMAGE_UUID)
-    create_and_activate(admin_client, 'networkService',
+    create_and_activate(internal_test_client, 'networkService',
                         networkServiceProviderId=ni.id,
                         networkId=network.id)
+    client = internal_test_client
```

Comment:
@pranavs18 I may have done a similar thing when we were troubleshooting a test, so I don't know if I gave you the wrong impression.  If I did that it was just since we were hacking the source to get things running